### ISSUE TITLE
autogen.sh on RHEL 7 fails. And documentation for the fix is difficult to find

### ISSUE THREAD

**Issue by calewis** [sstsimulator/sst-core#653]
New Issue for sst-core
----------------------
autogen.sh fails on RHEL 7.9

```
$ ./autogen.sh
Running /usr/bin/libtoolize...
libtoolize: `COPYING.LIB' not found in `/usr/share/libtool/libltdl'
libltdl doesn't exist.  Aborting.
```

2 - Describe how to reproduce

1. Clone the repo
2. Run `./autogen.sh` in the repo

3 - What Operating system(s) and versions 

RHEL 7.9

4 - What version of external libraries (Boost, MPI)
N/A

5 - Provide sha1 of all relevant sst repositories (sst-core, sst-elements, etc)
b77528dd9892d3f6d2bc18cbcf49b244ec28cb3a

**Comment by calewis** [comment]
This information should probably more prominent

The following is a bit convoluted: 
1. go to `http://sst-simulator.org`
2. click Documentation to get to http://sst-simulator.org/SSTPages/SSTMainDocumentation/
3. click button labeled "Detailed Installation Instructions" at the top of the page
4. Scroll down the page till seeing the link "SST Build Prerequisites"
5. Click the link
6. Finally down the page under GNU Libtool
>NOTE - On RedHat Linux type systems (including CentOS) you must also install the libtool-ltdl-devel package. This will provide the file COPYING.LIB needed by libtool.

**Comment by calewis** [comment]
PR #670 is an attempt to partially address this.

**Comment by calewis** [comment]
Fixed by #670
